portal/academy/models.py

```diff
@@ -11,11 +9,17 @@ class Specialization(models.Model):
     code = models.CharField(max_length=255, primary_key=True)
     name = models.CharField(max_length=255)
     description = models.TextField(blank=True)
-    created = models.DateTimeField(auto_now_add=True)
+    created = models.DateTimeField(editable=False)
 
     def __str__(self):
         return self.code
 
+    def save(self, *args, **kwargs):
+        ''' On save for the first time, set created timestamp'''
+        if not self.created:
+            self.created = timezone.now()
+        return super(Specialization, self).save(*args, **kwargs)
```

Comment:
No need to pass the class and the class instance to super.

Reply:
changed it accordingly :)